packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6481,6 +6481,48 @@ QUnit.module('valueChanged handler should receive correct event', {
     });
 });
 
+QUnit.module('onKeyDown', () => {
+    QUnit.test('should fire event only once (T1238121)', function(assert) {
+        const onKeyDownHandler = sinon.spy();
+        const products = [{
+            ID: 1,
+            Name: 'HD Video Player',
+            ImageSrc: 'https://js.devexpress.com/jQuery/Demos/WidgetsGallery/JSDemos/images/products/1-small.png',
+        }, {
+            ID: 2,
+            Name: 'SuperHD Player',
+            ImageSrc: 'https://js.devexpress.com/jQuery/Demos/WidgetsGallery/JSDemos/images/products/2-small.png',
+        }, {
+            ID: 3,
+            Name: 'SuperPlasma 50',
+            ImageSrc: 'https://js.devexpress.com/jQuery/Demos/WidgetsGallery/JSDemos/images/products/3-small.png',
+        }];
+        const $element = $('#selectBox').dxSelectBox({
+            dataSource: products,
+            displayExpr: 'Name',
+            valueExpr: 'ID',
+            value: products[0].ID,
```

Comment:
do we need all these configurations to test it? probably minimal configuration will work, no need to add redundant options and dataSource data

Reply:
[889c03f](https://github.com/DevExpress/DevExtreme/pull/27660/commits/889c03f35413c3be25904a85b8640bb2a203714f)